applications/Colossal-LLaMA/colossal_llama/dataset/conversation.py

```diff
@@ -100,7 +100,7 @@ def dict(self):
     messages=[],
     offset=0,
     sep_style=SeparatorStyle.ADD_BOS_EOS_TOKEN,
-    seps=["<|begin_of_text|>", "<|end_of_text|>"],
+    seps=["<|begin_of_text|>", "<|eot_id|>"],
```

Comment:
@TongLi3701 Please take a look. Is this right?

Reply:
It should be ok for now.